pleskdistup/common/src/log.py

```diff
@@ -9,10 +10,20 @@ class logger():
 
     is_streams_enabled = False
     streams_logger = logging.getLogger("distupgrade_streams")
+    encoding: str = locale.getpreferredencoding()
+
+    @staticmethod
+    def _re_decode_message(message: str) -> str:
+        return str(message).encode(logger.encoding, errors='backslashreplace').decode(logger.encoding, errors='backslashreplace')
```

Comment:
According to annotation, `message` should be also `str`, why `str()` then?

Reply:
We sometimes send objects into log functions, like in this [example] (https://github.com/plesk/dist-upgrader/blob/main/pleskdistup/main.py#L111) , where a failed pre-check is logged to the error log.
Likely we could just change type annotation to "something convertible to string"

Reply:
Ah, wait a second, the place was actually broken by this change, we just need to fix it